### PR TITLE
Patch release v0.60.1

### PR DESCRIPTION
**Pull request by chitrangpatel**
Updates to release.md

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Further documentation available:
 - **Latest Release**: [v0.60.0][v0.60-0] (2024-05-22) ([docs][v0.60-0-docs], [examples][v0.60-0-examples])
```
Comment (JeromeJu) [inline releases.md:44]: This might also needs to be v0.60.1

File: releases.md [hunk 2]
```diff
@@ -275,6 +276,7 @@ Older releases are EOL and available on [GitHub][tekton-pipeline-releases].
 [v0-37-0]: https://github.com/tektoncd/pipeline/releases/tag/v0.37.0
 
 [v0.60-0-docs]: https://github.com/tektoncd/pipeline/tree/v0.60.0rdocs#tekton-pipelines
```
Comment (JeromeJu) [inline releases.md:278]: nit: Just realized `r/ should be replaced by '/'`

**Review by JeromeJu — APPROVED** [review]

File: releases.md (hunk 1 above)
Comment (chitrangpatel) [reply releases.md:44]: Fixed

File: releases.md (hunk 2 above)
Comment (chitrangpatel) [reply releases.md:278]: Fixed

**Pull request merged**